kernel/entries.ml

```diff
@@ -99,10 +99,13 @@ type projection_entry = {
   proj_entry_ind : MutInd.t;
   proj_entry_arg : int }
 
+type primitive_entry = types in_constant_universes_entry * CPrimitives.op_or_type
```

Comment:
>Right, what is the recommended way of doing that?

Here instead of `in_constant_universes_entry` you would use `in_universe_context_set` maybe with a comment `(* always monomorphic *)`, then the typing errors should guide you.